dv/common/dv_common.core

```diff
@@ -0,0 +1,22 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:common"
+description: "Collection of useful RTL for building simulations"
+filesets:
+  files_sim_sv:
+    files:
+      - rtl/prim_clock_gating.sv
+      - rtl/ram_1p.sv
+      - rtl/bus.sv
+      - rtl/simulator_ctrl.sv
+      - cpp/sim_output_manager.h:  { file_type: cppSource, is_include_file: true }
+      - cpp/sim_output_manager.cc: { file_type: cppSource }
```

Comment:
You need to split this file to not include files which might not be needed in certain cases. So that would remove the C++ files at least.